lib/api/model/events.dart

```diff
@@ -265,6 +267,89 @@ class MessageEvent extends Event {
   }
 }
 
+/// A Zulip event of type `update_message`.
+@JsonSerializable(fieldRename: FieldRename.snake)
+class UpdateMessageEvent extends Event {
+  @override
+  String get type => 'update_message';
+
+  final int? userId; // TODO(server-5)
+  final bool? renderingOnly; // TODO(server-5)
+  final int messageId;
+  final List<int> messageIds;
+  final List<String> flags;
+  final int? editTimestamp; // TODO(server-5)
+  final String? streamName;
+  final int? streamId;
+  final int? newStreamId;
+  final String? propagateMode;
```

Comment:
```suggestion
  final String? propagateMode; // TODO enum
```

Or maybe just make the enum.